OData V2 / V4 with feature toggles (#1234)
cap-js/docs: java/migration.md

	```

	This command picks up all service definitions in the Java project base directory (`srv` by default) and generates EDMX for OData V2. It also localizes the generated EDMX files with all available translations. For more information on the previous command, call `cds help compile` on the command line. If your service definitions are located in a different directory, adopt the previous command. If your service definitions are contained in multiple directories, add the previous command for each directory separately. Make sure to use at least `cds-dk 3.2.0` for this step.

> If you are using feature toggles in your CAP Java project, you have to specify the features root folder as-well in the command line:

Reviewer: Maybe:

```suggestion
If you are using feature toggles in your CAP Java project, the list of models must also contain the features' root folder:
```

	```xml
	<command>compile ${project.basedir} ${session.executionRootDirectory}/fts/* -s all -l all -2 edmx-v2 -o ${project.basedir}/src/main/resources/edmx/v2</command>
	```

> 	This command includes the folder `/fts` and all sub-folders into the CDS model.

Reviewer: ```suggestion
	This command includes the folder _/fts_ and all sub-folders into the CDS model.
```

I think we use _italics_ for file and folder names.
